src/MusicStore/Controllers/ManageController.cs

```diff
@@ -119,25 +116,25 @@ public IActionResult AddPhoneNumber()
         // POST: /Manage/DisableTwoFactorAuthentication
         [HttpPost]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> DisableTwoFactorAuthentication()
+        public async Task<IActionResult> DisableTwoFactorAuthentication(CancellationToken cancellationToken)
```

Comment:
Don't have a particularly strong opinion on this. That said, the param name `cancellationToken` is ubiquitous in all APIs and in theory the request being aborted is not the only reason you might want to pass in a CancellationToken to an application.